### ISSUE TITLE
Problems using ‘/image’ to send AVIF, PNG, and WebP images

### ISSUE THREAD

**Issue by kseistrup** [simplex-chat/simplex-chat#924]
I attempted to send the same image as AVIF, JPEG, PNG and WebP from the desktop app using ther `/image` command. Only the JPEG image was sent. For the AVIF, PNG and WebP formats SimpleX said:

> max image size: 236700 bytes, resize it or send as a file using /f

although the sizes were 9,251 bytes, 68,929 bytes and 9,738 bytes respectively.

I could send the images with the `/file` command, but then they wouldn't show up inline in the mobile app — not even the PNG. Instead they would just be available for download, with no indication that they were image files.

I was using SimpleX v3.1 as desktop app and mobile app. The mobile is fairly recent and was able to display all the mentioned image formats inline in other apps.

I've attached all images in a gzipped tarball as Microsoft Giðub supports neither WebP, nor AVIF upload.

:paperclip: [www_karussel_ventetider.tar.gz](https://github.com/simplex-chat/simplex-chat/files/9299027/www_karussel_ventetider.tar.gz)

**Comment by epoberezkin** [comment]
Yep, it's currently indeed limited to JPG support - something to improve...

**Comment by spirillen** [comment]
I would vote for the ability to send valid images formats as images.
